<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillStatsWidget</class>
 <widget class="QWidget" name="SkillStatsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="hLayoutMain">
   <item>
    <widget class="QLabel" name="labelCounter">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayoutSkillInfo"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTripod1">
     <property name="title">
      <string>트라이포드 I</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="hLayoutTripod1"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTripod2">
     <property name="title">
      <string>트라이포드 II</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="hLayoutTripod2"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTripod3">
     <property name="title">
      <string>트라이포드 III</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="hLayoutTripod3"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupRune">
     <property name="title">
      <string>룬</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="hLayoutRune"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
